app/controllers/concerns/sufia/works_controller_behavior.rb

```diff
@@ -72,6 +72,14 @@ def after_update_response
         end
       end
 
+      def after_destroy_response(title)
+        flash[:notice] = "Deleted #{title}"
+        respond_to do |wants|
+          wants.html { redirect_to sufia.dashboard_works_path }
```

Comment:
As written, the `flash[:notice]` will be set even for JSON responses. You can move the `flash[:notice]` to `redirect_to` command via the following:

```ruby
wants.html { redirect_to sufia.dashboard_works_path, notice: "Deleted #{title}" }
```